### PR TITLE
typings(Popup, Menu): Fixed Popup inverted and MenuItem content types

### PR DESCRIPTION
**Pull request by dstpierre**

### CODE REVIEW

File: src/collections/Menu/index.d.ts [hunk 1]
```diff
@@ -135,7 +135,7 @@ interface MenuItemProps extends ReactMouseEvents<HTMLElement> {
   color?: SemanticCOLORS;
 
   /** Shorthand for primary content. */
-  content?: string;
+  content?: any;
```
Comment (levithomason) [inline src/collections/Menu/index.d.ts]: This is actually `PropTypes.node` which is:

>Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.

https://facebook.github.io/react/docs/typechecking-with-proptypes.html

**Review by levithomason — CHANGES_REQUESTED** [review]

File: src/collections/Menu/index.d.ts (hunk 1 above)
Comment (levithomason) [reply src/collections/Menu/index.d.ts]: Per @layershifter in https://github.com/Semantic-Org/Semantic-UI-React/pull/1159#issuecomment-272907848, let's use `React.Node`.

**Comment by dstpierre** [comment]
I used `React.ReactNode` instead of the `any`.

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Thanks much!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.1`.

**Pull request merged**